src/manifest.json

```
@@ -0,0 +1,53 @@
+{
+    "app_id": "pagerduty",
+    "version": "v1.0.0",
```

Let's be consistent with the other apps
```suggestion
    "version": "1.0.0",
```